compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -15747,29 +15772,19 @@ OMR::Z::TreeEvaluator::arraycmpSIMDHelper(TR::Node *node,
    generateRXInstruction(cg, TR::InstOpCode::getLoadAddressOpCode(), node, secondAddrReg, generateS390MemoryReference(secondAddrReg, 16, cg));
 
    // eds : perf : Can use BRXLE with negative increment for this
```

Comment:
@r30shah I attempted to make use of BRXLE as this comment suggest but didn't finish since it was out of scope of this PR. Let me know if you think it worth the effort.

Reply:
Agree, it is beyond the scope of this PR. Also, before even attempting to use BRXLE, I would write down the sequence of instruction we will be generating with BRXLE and see if there is any benefit of combining two ops here,